src/pretix/base/metrics.py

```diff
@@ -0,0 +1,160 @@
+from django.conf import settings
+
+if settings.HAS_REDIS:
+    import django_redis
+
+REDIS_KEY_PREFIX = "pretix_metrics_"
+
+r = django_redis.get_redis_connection("redis")
```

Comment:
`rc`, up until now, but it wasn't used as a global variable. Perfect time and location to start a good, new convention ;)